src/jsx/relax.jsx

```diff
@@ -541,6 +541,7 @@ class RELAXContents extends React.Component {
 
         <div id="branch-attribute-table">
           <Header title="Branch attributes" />
+          <div style={{ height: 10 }}></div>
```

Comment:
Seems strange not to use margin. It looks like this is a pervasive `<Header />` component issue?